src/rename_rules.rs

```diff
@@ -21,7 +21,7 @@ pub fn rename_by_rule(input: &str, rule: &str) -> String {
         "SCREAMING_SNAKE_CASE" => input.to_shouty_snake_case(),
         "kebab-case" => input.to_kebab_case(),
         "SCREAMING-KEBAB-CASE" => input.to_shouty_kebab_case(),
-        _ => input.to_string(),
+        data => data.to_string(),
```

Comment:
Hmm, why `data => data.to_string()`? We match `rule`, so this `data` would be `rule` as well, but we want to just use `input` as-is here. Don't we?

Reply:
It might be the wrong place to make a change. Take a look into the linked issue.
Earlier I could rename the command. Unfortunately, it seems not working right now. 
I want to type something:

...
enum BlahBlah {
#[command(rename = "ab")]
Command1
}

so it should react to ab, but in current implementation it would still work as Command1